### ISSUE TITLE
Daily Streak Broken

### ISSUE THREAD

**Issue by FredTheFrog21** [DankMemer/dank-memer-suggestions-and-bugs#660]
So, basically when I do pls daily, I only ever get 1,500 and the daily Streak never goes up...This has been happening for the past 3 days, it's really annoying me as I am trying to do the dragon quest and have done everything but the 7 day Streak which I can't even do.

**Comment by melmsie** [comment]
Need your discord ID

**Comment by FredTheFrog21** [comment]
My tag? It's @FredTheFrog21#9511

**Comment by OmniTraderX** [comment]
I'm actually having this same problem as well.

**Comment by OmniTraderX** [comment]
JTW (🤖📈👾📊)#9710 is my Discord ID. I can change it if that's a pain in the but to deal with. lol

**Comment by melmsie** [comment]
That is still not a Discord ID.

https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-

@FredTheFrog21 @OmniTraderX

**Comment by OmniTraderX** [comment]
608727754627874871
There's mine. Thanks @melmsie

**Comment by ooftheloofer** [comment]
This bug has been annoying me so much too. Im trying to get a dragon, meaning I need a 7 day streak, but since its stuck at 0, I CANT DO ANYTHING 😫 My discord id is ooftheloofer#6482

**Comment by ooftheloofer** [comment]
Oh sorry, I sent my tag. 347180409047744512 is my ID

**Comment by ooftheloofer** [comment]
I did a 3 day streak so far, jusg saying so in case you fix this, I can continue my streak

**Comment by KennywasTaken** [comment]
@melmsie I have the same daily streak issue my discord id is 400855350045048833

**Comment by FredTheFrog21** [comment]
Oh my bad @melmsie I didn't know what you meant about discord ID, I thought you meant my tag. But thanks to that link you sent, I realise my ID is 615301317459574794

**Comment by FredTheFrog21** [comment]
Let me know once you have done what ever is being done, thanks for your time @melmsie

**Comment by OmniTraderX** [comment]
Have we seen any updates on this? Just checked my daily streak and it's still at 0. @melmsie

**Comment by ThisIsARealNamez** [comment]
@melmsie I've claimed my daily for 3 days so far and the streak is still at 0. My Discord ID is 476418499615391764

**Comment by FredTheFrog21** [comment]
Hey so uh @melmsie is there a way to do this or not because I'm really bummed, today is the 6th day it hasn't had the streak and I feel I'm never gonna get the dragon

**Comment by ooftheloofer** [comment]
Has there been any update so far? I just did my 6th day and still 0. Guess I cant get dragon yet

**Comment by ooftheloofer** [comment]
Ive done my 7 day of daily streak, meaning I can get the dragon now, but its still at 0. Any progress @melmsie

**Comment by VGsBlaze** [comment]
Yeah I’m stuck at A 0 streak When I’ve been using pls daily for the past 3 weeks or so. My ID is 513064251716403211
